app/models/person.rb

```diff
@@ -15,6 +15,5 @@ class Person < ApplicationRecord
   def search_for_missing_person
     []
```

Comment:
Not sure what this is doing here. It's not assigned to any variable, is it just a typo?